cmd/ip-masq-agent/ip-masq-agent_test.go

```diff
@@ -82,7 +82,7 @@ var syncConfigTests = []struct {
 	cfg  *MasqConfig       // expected values of the configuration after loading from fs
 }{
 	// valid yaml
-	{"valid yaml file, all keys", fakefs.StringFS{`
+	{"valid yaml file, all keys", fakefs.StringFS{File: `
```

Comment:
Thanks for fixing these :)

Reply:
No problem! I believe `go vet` was complaining about these in Go 1.11.